include/common/mpmc_queue.h

```diff
@@ -28,48 +28,36 @@ namespace grppi{
 
 constexpr int DEFAULT_SIZE = 100;
 
+
+
 template <typename T>
 class mpmc_queue{
    private:
       int size;
       std::vector<T> buffer;
+      Queue_mode lockfree = Queue_mode::blocking;
```

Comment:
Queue_mode has been renamed to queue_mode and the member variable has been renamed to mode.